src/dashboard/luasrc/controller/ids-priv/ids_tab.lua

```diff
@@ -22,7 +22,7 @@ module("luci.controller.ids-priv.ids_tab", package.seeall)  --notice that new_ta
         local payload = luci.jsonc.parse(tostring(config))
```

Comment:
Recognizing that you made edits to the luci code you should probably wait until Lucas' PR is merged, then pull and resubmit this PR since he does have a few edits in his branch that you replicate in yours.